app/backend/src/couchers/models.py

```diff
@@ -269,6 +272,14 @@ def mutual_friends(self, target_id):
             .all()
         )
 
+    def blocked_blocking_users(self):
+        relevant_blocks = []
+        for userblock in self.is_blocking_user:
+            relevant_blocks.append(userblock.blocked_user_id)
+        for userblock in self.is_blocked_user:
+            relevant_blocks.append(userblock.blocking_user_id)
+        return relevant_blocks
```

Comment:
I'll add a comment, but basically, for the most part, this should return everyone involved in a block with you so that we can filter for them (i.e. session.query(Users).filter(request.user is not in context.user.blocked_blocking_users)